web/packages/teleport/src/Discover/Database/TestConnection/TestConnection.story.tsx

```diff
@@ -54,4 +62,5 @@ const props: State = {
     users: ['user1', 'user2'],
     hostname: 'db-hostname',
   },
+  dbEngine: DatabaseEngine.MySQL,
```

Comment:
same comment as above re: default `dbEngine: null` and always pass it. Feel free to disregard tho